src/common/utility.ts

```diff
@@ -458,3 +458,48 @@ export const handleRedirect = (residence, current_client_country) => {
 export const isLocalhost = () => !!(isBrowser() && process.env.NODE_ENV === 'development')
 
 export const isTestlink = () => !!(isBrowser() && window.location.hostname.includes('binary.sx'))
+
+export const getSocialMediaUrl = (type: string, is_eu = false, is_uk = false): string => {
```

Comment:
Can we call `is_eu` and `is_uk` inside the function instead of passing it as a parameter?

Reply:
@prince-deriv no. When we call useCountryRule here we can't see any page. Can't figure out why, but I'll try again

Reply:
Done